src/components/TextButton.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import styled, { CSSProperties } from 'styled-components'
+
+export interface textButtonInterface {
+    text: string
```

Comment:
I thought about this too, and wondered what else we might want to put in a link. I think we have enough TextButtons though to warrant its own component. Maybe in another ticket, if we feel we need to, we can make a generic link that takes anything, and then we can change TextButton to extend that with its own defaults. 

I say this as well because the functionality of the TextButton (underline on hover etc.) is specific to text and not relative to a generic case. This would lead to a bloated component in the future I think if we tried to make something work for everything without some kind of inheritance scheme.

Reply:
So should I keep it as is for now?

Reply:
Yes, I think keeping it as is for now would be good.  If we find that we want the TextButton to also include icons or be more general we can make another ticket for it.  The idea would be to have a generic button with prop `type` with options such as `{success, warning, main, disabled, link}` where each type has a different css styling associated with it such as colour or underlined text.

Reply:
Sounds rad! Once it comes up I'd love to work on that :^)